src/Command.js

```diff
@@ -19,6 +19,47 @@ export default class Command {
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
   }
 
+  get name() {
+    // For a class named "FooCommand" this returns "foo".
+    return this.className.replace("Command", "").toLowerCase();
+  }
+
+  get className() {
+    return this.constructor.name;
+  }
+
+  // Override this to inherit config from another command.
+  // For example `updated` inherits config from `publish`.
+  get otherCommandConfigs() {
+    return [];
+  }
+
+  getOptions(...objects) {
+
+    // Items lower down override items higher up.
+    return Object.assign(
+      {},
+
+      // Deprecated legacy options in `asini.json`.
+      this._legacyOptions(),
+
+      // Global options from `asini.json`.
+      this.repository.asiniJson,
+
+      // Option overrides for commands.
+      // Inherited options from `otherCommandConfigs` come before the current
+      // command's configuration.
+      ...[...this.otherCommandConfigs, this.name]
+        .map((name) => ((this.repository.asiniJson || {}).command || {})[name]),
+
+      // For example, the item from the `packages` array in config.
+      ...objects,
+
+      // CLI flags always override everything.
+      this.flags,
+    );
```

Comment:
i think the Flags section of the README need an entire section talking about precedence.